scene/gui/graph_node.h

```diff
@@ -31,170 +31,113 @@
 #ifndef GRAPH_NODE_H
 #define GRAPH_NODE_H
 
-#include "scene/gui/container.h"
-#include "scene/resources/text_line.h"
+#include "scene/gui/graph_element.h"
 
-class GraphNode : public Container {
-	GDCLASS(GraphNode, Container);
+class HBoxContainer;
 
-	struct _MinSizeCache {
-		int min_size;
-		bool will_stretch;
-		int final_size;
-	};
-
-public:
-	enum Overlay {
-		OVERLAY_DISABLED,
-		OVERLAY_BREAKPOINT,
-		OVERLAY_POSITION
-	};
+class GraphNode : public GraphElement {
+	GDCLASS(GraphNode, GraphElement);
 
 private:
```

Comment:
I thought this was somewhat code-style related. We do this is many classes (e.g. Label, Button) so I suggest keeping it for now. If we decide that the redundant `private` modifier should be removed, then I would do it for in a cleanup PR for all classes.

Reply:
I haven't checked those classes, but we don't normally add private if it's the first block. Here it's there because there used to be a block with a different scope above it.